lib/data.dart

```diff
@@ -0,0 +1,66 @@
+class Category {
```

Comment:
This whole data file is a very cool concept, just wanted to emphasize that I'm going to steal it for my next laboratory works, thank you very much